libs/sync/src/lib/sync-details-page/sync-modal/sync-modal.component.html

```diff
@@ -0,0 +1,20 @@
+<pa-modal-dialog>
+  <pa-modal-title>
+    <h2>{{ 'sync.details.modal.title' | translate }}</h2>
+  </pa-modal-title>
+  <pa-modal-content>
+    <div>{{ 'sync.details.modal.description' | translate }}</div>
+    <div class="actions">
```

Comment:
shouldn't those buttons be in `pa-modal-footer` instead of directly in the modal content?